client/my-sites/checkout/checkout-thank-you/premium-plan-details.jsx

```diff
@@ -59,13 +64,34 @@ const PremiumPlanDetails = ( { selectedSite, sitePlans, selectedFeature, purchas
 			/>
 
 			<QuerySiteVouchers siteId={ selectedSite.ID } />
-			<div>
-				<GoogleVoucherDetails selectedSite={ selectedSite } />
-			</div>
+			<PurchaseDetail
+				id="google-credits"
+				icon={
+					<img
+						alt={ i18n.translate( 'Google AdWords' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Setup Google AdWords' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).